server.js

```diff
@@ -25,11 +65,57 @@ const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to my mongoDB API!')
```

Comment:
Although this greeting is just fine i quip tip on a package I used is list endpoints from express-list-endpoints which offers en simple guide over avaiable paths in the api